script.js

```diff
@@ -0,0 +1,165 @@
+// Global Elements
+let scoreGrid = [['','',''] ,['','',''], ['','','']];
+let playerOneTurn = true;
+let bot = false;
+
+// Dom Elements
+const colorMode = document.getElementById('color-mode');
+const gameMode = document.getElementById('game-mode');
+const gameGrids = document.querySelectorAll('.grid-space');
+const message = document.getElementById('message');
+const gridTable = document.querySelector('.grid-table');
+
+
+let currentColor =  'dark';
+
+
+const insertInScoreGrid = (current,pos,symb) => {
+    current.innerText = symb;
+    const [one, two] = get2DPosition(pos);
```

Comment:
I can tell what's going on, but in general you want to name your variables something that communicates more clearly to the reader what's going on. For example, x&y, or vert&horiz, or row&col